src/test/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealServiceTest.java

```diff
@@ -209,6 +240,36 @@ public void shouldSendRoboticsByEmail() {
         then(emailService).should(times(2)).sendEmail(any(Email.class));
     }
 
+    @Test
+    public void shouldSendRoboticsByEmailPdfOnly() {
+
+        byte[] expected = {};
+
+        given(pdfServiceClient.generateFromHtml(any(byte[].class), any(Map.class))).willReturn(expected);
+
+        submitAppealService.submitAppeal(appealData);
+
+        verify(emailService, times(2)).sendEmail(emailCaptor.capture());
+        Email roboticsEmail = emailCaptor.getAllValues().get(1);
+        List<EmailAttachment> attachments = roboticsEmail.getAttachments();
```

Comment:
attachemtns is not used, please, use it or delete it :)

Reply:
This test is a duplicate of this one:  shouldSendRoboticsByEmail(). 

Please, remove this one  shouldSendRoboticsByEmail() as it is covered in your test.